### PR TITLE
fix(btp-utils): fix getCredentialsForDestinationService

### PR DESCRIPTION
**Pull request by zdravko-georgiev**
Issue: https://github.com/SAP/open-ux-tools/issues/546

### CODE REVIEW

File: packages/btp-utils/src/app-studio.ts [hunk 1]
```diff
@@ -34,9 +34,9 @@ export function getAppStudioProxyURL(): string | undefined {
  */
 export async function getCredentialsForDestinationService(instance: string): Promise<string> {
     try {
-        const serviceInfo = await cfGetInstanceKeyParameters(instance);
-        const clientId = serviceInfo.uaa?.clientid || serviceInfo.clientid;
-        const clientSecret = serviceInfo.uaa?.clientsecret || serviceInfo.clientsecret;
+        const serviceCredentials = (await cfGetInstanceKeyParameters(instance)).credentials;
```
Comment (tobiasqueck) [inline packages/btp-utils/src/app-studio.ts:37]: Is it possible that `await cfGetInstanceKeyParameters(instance)` returns `undefined` or `null`?

**Review by tobiasqueck — DISMISSED** [review]
Small comment, if a null check is not required, then all is good.

File: packages/btp-utils/src/app-studio.ts (hunk 1 above)
Comment (zdravko-georgiev) [reply packages/btp-utils/src/app-studio.ts:37]: Yes, it is possible https://github.com/SAP/cloud-foundry-tools-api/blob/master/src/cf-local.ts#L592. I will update the PR.

Comment (zdravko-georgiev) [reply packages/btp-utils/src/app-studio.ts:37]: @tobiasqueck On second thought, if it returns `undefined`, then an error will be thrown that we can get credentials. Which is OK, right? I mean the whole flow can't work without credentials AFAIK.

Comment (zdravko-georgiev) [reply packages/btp-utils/src/app-studio.ts]: Done with https://github.com/SAP/open-ux-tools/pull/547/commits/e1d0434d06b5a6cb816f3de2313a8892f58ab088

**Review by tobiasqueck — APPROVED** [review]

**Review by tobiasqueck — APPROVED** [review]

**Review by Klaus-Keller — APPROVED** [review]
Code looks good, test exists, approved.

**Pull request merged**